jetty-http2/http2-common/src/main/java/org/eclipse/jetty/http2/HTTP2Cipher.java

```diff
@@ -20,323 +20,312 @@
-            "TLS_PSK_WITH_ARIA_256_GCM_SHA384",
-            "TLS_RSA_PSK_WITH_ARIA_128_GCM_SHA256",
-            "TLS_RSA_PSK_WITH_ARIA_256_GCM_SHA384",
-            "TLS_ECDHE_PSK_WITH_ARIA_128_CBC_SHA256",
-            "TLS_ECDHE_PSK_WITH_ARIA_256_CBC_SHA384",
-            "TLS_ECDHE_ECDSA_WITH_CAMELLIA_128_CBC_SHA256",
-            "TLS_ECDHE_ECDSA_WITH_CAMELLIA_256_CBC_SHA384",
-            "TLS_ECDH_ECDSA_WITH_CAMELLIA_128_CBC_SHA256",
-            "TLS_ECDH_ECDSA_WITH_CAMELLIA_256_CBC_SHA384",
-            "TLS_ECDHE_RSA_WITH_CAMELLIA_128_CBC_SHA256",
-            "TLS_ECDHE_RSA_WITH_CAMELLIA_256_CBC_SHA384",
-            "TLS_ECDH_RSA_WITH_CAMELLIA_128_CBC_SHA256",
-            "TLS_ECDH_RSA_WITH_CAMELLIA_256_CBC_SHA384",
-            "TLS_RSA_WITH_CAMELLIA_128_GCM_SHA256",
-            "TLS_RSA_WITH_CAMELLIA_256_GCM_SHA384",
-            "TLS_DH_RSA_WITH_CAMELLIA_128_GCM_SHA256",
-            "TLS_DH_RSA_WITH_CAMELLIA_256_GCM_SHA384",
-            "TLS_DH_DSS_WITH_CAMELLIA_128_GCM_SHA256",
-            "TLS_DH_DSS_WITH_CAMELLIA_256_GCM_SHA384",
-            "TLS_DH_anon_WITH_CAMELLIA_128_GCM_SHA256",
-            "TLS_DH_anon_WITH_CAMELLIA_256_GCM_SHA384",
-            "TLS_ECDH_ECDSA_WITH_CAMELLIA_128_GCM_SHA256",
-            "TLS_ECDH_ECDSA_WITH_CAMELLIA_256_GCM_SHA384",
-            "TLS_ECDH_RSA_WITH_CAMELLIA_128_GCM_SHA256",
-            "TLS_ECDH_RSA_WITH_CAMELLIA_256_GCM_SHA384",
-            "TLS_PSK_WITH_CAMELLIA_128_GCM_SHA256",
-            "TLS_PSK_WITH_CAMELLIA_256_GCM_SHA384",
-            "TLS_RSA_PSK_WITH_CAMELLIA_128_GCM_SHA256",
-            "TLS_RSA_PSK_WITH_CAMELLIA_256_GCM_SHA384",
-            "TLS_PSK_WITH_CAMELLIA_128_CBC_SHA256",
-            "TLS_PSK_WITH_CAMELLIA_256_CBC_SHA384",
-            "TLS_DHE_PSK_WITH_CAMELLIA_128_CBC_SHA256",
-            "TLS_DHE_PSK_WITH_CAMELLIA_256_CBC_SHA384",
-            "TLS_RSA_PSK_WITH_CAMELLIA_128_CBC_SHA256",
-            "TLS_RSA_PSK_WITH_CAMELLIA_256_CBC_SHA384",
-            "TLS_ECDHE_PSK_WITH_CAMELLIA_128_CBC_SHA256",
-            "TLS_ECDHE_PSK_WITH_CAMELLIA_256_CBC_SHA384",
-            "TLS_RSA_WITH_AES_128_CCM",
-            "TLS_RSA_WITH_AES_256_CCM",
-            "TLS_RSA_WITH_AES_128_CCM_8",
-            "TLS_RSA_WITH_AES_256_CCM_8",
-            "TLS_PSK_WITH_AES_128_CCM",
-            "TLS_PSK_WITH_AES_256_CCM",
-            "TLS_PSK_WITH_AES_128_CCM_8",
-            "TLS_PSK_WITH_AES_256_CCM_8"
-            */
-        };
-        _xxx = ciphers;
-        for (String c : ciphers)
-        {
-            __blackCiphers.put(c, Boolean.TRUE);
-        }
-    }
+    private static final Index<Boolean> __blackCiphers = new Index.Builder<Boolean>()
+        .caseSensitive(false)
+        .with("TLS_NULL_WITH_NULL_NULL", Boolean.TRUE)
+//        .with("TLS_RSA_WITH_NULL_MD5", Boolean.TRUE)
+        .with("TLS_RSA_WITH_NULL_SHA", Boolean.TRUE)
+        .with("TLS_RSA_EXPORT_WITH_RC4_40_MD5", Boolean.TRUE)
+        /*
```

Comment:
oops these have been accidentally commented out!

Reply:
... I think by me!!!!

Reply:
Well, I don't know what you did, but you did it in https://github.com/eclipse/jetty.project/commit/0a70cba8ae8ecb9d770b7ea03987e650c90b0859

Reply:
I think I did a merge from 9 to 10 in the branch I was experimenting!  My bad!